### ISSUE TITLE
Translating Surveys. 

### ISSUE THREAD

**Issue by ekowkaay** [SalesforceLabs/survey-force#135]
Hi there, 

We heavily use survey force, and would like to translate our surveys to about 25 languages, I was wondering if there's a better to go about this process. Any thoughts would be much appreciated.

**Comment by jrattanpal** [comment]
Survey questions are data. There is no easy option to translate data. 

You could look into creating a translate option but you will have to build it yourself. Because Survey Force is un-managed package, you own the code and change it to fit your needs.

**Comment by ekowkaay** [comment]
So since it's data like you said , can we just create the template in a specific language?

**Comment by jrattanpal** [comment]
Certainly. When you add questions, you can use any language you want as it's simple input text. 

Or at least I think it should work. If some characters don't work then let me know (With example text as well as screenshot). I'll see what needs to be done to make sure proper language is used. Although, I think it should work.

You can test and let me know

**Comment by ekowkaay** [comment]
Will do! Thanks so much for the clarification.
